en/config_vtol/README.md

```diff
@@ -1,3 +1,11 @@
 # VTOL Configuration
 
-This section contains topics related to VTOL configuration and tuning.
\ No newline at end of file
+First perform the [Standard Configuration](../config/README.md).
+As part of this you should calibrate the [Airspeed sensor](../config/airspeed.md) (optional, but highly recommended) and [assign a VTOL transition switch](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set) to your RC controller.
+
+Then perform VTOL-specific configuration and tuning:
+
+* [QuadPlane Configuration](../config_vtol/vtol_quad_configuration.md)

```

Comment:
@sfuhrer IMO this is actually "Standard Plane" configuration - right - ie it doesn't have to be a quad plane, it is just that it's a multicopter grafted onto a fixed wing?

Reply:
Do you mean "Standard VTOL"? TBH I don't like "Standard VTOL" too much, but as you say, QuadPlane doesn't work to be a generic name of this type as there can be more than 4 hover motors. And yes, the category defines itself with having separate propulsion systems for hover and cruise flight (opposed to tilt-body (aka tailsitter), tilt-wing, tilt-rotor). As I don't have a better proposal for this category we should stick to "Standard VTOL" consistently.

Reply:
In "Standard Plane" I would miss the VTOL notion - but maybe "Standard Plane VTOL" to make it clear that it has a wing (opposed to all multirotors or helicopters that actually are just as much VTOLs). This categorization is really not easy.

Reply:
The first two are mainly for standard VTOLs. 

I'd propose to have the following subcategories in VTOL Config/Tuning:
- Transition tuning
    - Standard VTOL
    - Tiltrotor
    - Tailsitter
- weather vane
- Safety (in here we can talk about Quadchutes, permanent stabilization, ..)

Reply:
Sorry, yes I do mean Standard VTOL. Tired! I've used that elsewhere. I'd refer to QuadPlane for the Quad motor Standard VTOL. Yes, it is a crap name, but I don't have anything better either.

Reply:
I am happy enough with that organisation, but I need to map it to the docs we have.

So I think you are saying
- Transition tuning
   - Standard VTOL
      * [QuadPlane Configuration](../config_vtol/vtol_quad_configuration.md)
      * [Back-transition Tuning](../config_vtol/vtol_back_transition_tuning.md)
   - Tiltrotor
      - Nothing?
   - Tailsitter
      - Nothing?
- weather vane
   - [VTOL Weather Vane](../config_vtol/vtol_weathervane.md)
- Safety

What about this one: * [VTOL w/o Airspeed Sensor](../config_vtol/vtol_without_airspeed_sensor.md)

Reply:
Ignore. I see you're going to PR a structural update.